### ISSUE TITLE
Split the annotate view into two (free text and structured annotate) views?

### ISSUE THREAD

**Issue by JulianKniephoff** [opencast/annotation-tool#129]
Once we have #113, why should we use a different mechanism to configure the visibility of the subcomponents of the annotate view?

**Comment by ebbertd** [comment]
I don't think it is needed to have the free text annotate and the structure annotate views split into two. If both are needed it makes sense to have them together. As long as it is possible to turn them off / on using the *View* menu.

**Comment by JulianKniephoff** [comment]
Alright, noted. However, I think this change would actually even simplify the code **and** give us a new capability (that we could also easily disable if it is unwanted), which makes it hard to resist. :wink:
